src/peer/socket.js

```diff
@@ -169,58 +169,58 @@ class Socket extends EventEmitter {
 
       http.timeout = config.DISPATCHER_TIMEOUT;
       http.open('GET', this._dispatcherUrl, true);
-
       /* istanbul ignore next */
       http.onerror = () => {
-        reject(new Error('There was a problem with the dispatcher.'));
+        reject(
+          new Error(
+            'There was a problem with the request for the dispatcher. Check your request and network connections.'
+          )
+        );
       };
 
       http.onabort = () => {
-        reject(new Error('The request was aborted.'));
+        reject(new Error('The request for the dispatcher was aborted.'));
       };
 
       http.ontimeout = () => {
-        reject(new Error('The request for the dispather timed out.'));
+        reject(
+          new Error(
+            'The request for the dispatcher timed out. Check your firewall, network speed, Skyway failure information'
```

Comment:
`Skyway` should be `SkyWay`.

BTW the term `SkyWay` is for Japan only, it may be `ECLWebRTC` for non-Japanese, even it's better to omit this..?

Reply:
I inquired about this in SkyWay team. And `SkyWay` will have no trouble.
So this code use `SkyWay`. Thank you.